content/quick-start.mdx

```diff
@@ -100,7 +98,45 @@ npm run dev
 This will start the development server. 
 Now, open your browser and navigate to `localhost:3000` to see your application running.
 
-### Explore more templates
+#### Explore more templates
```

Comment:
I think this better fits an aside since it's kinda like extra info on templates and where to find them

Reply:
asides will be used as callouts for additional content that are tangentially related vs added content. 

they should also be used sparingly - in this instance it's better to stay as a heading vs aside!